config/src/test/java/org/axonframework/config/testsuite/student/MultiStateCommandHandlingComponentTest.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2010-2025. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.config.testsuite.student;
+
+
+import org.axonframework.config.testsuite.student.commands.EnrollStudentToCourseCommand;
+import org.axonframework.config.testsuite.student.events.StudentEnrolledEvent;
+import org.axonframework.config.testsuite.student.state.Course;
+import org.axonframework.config.testsuite.student.state.Student;
+import org.axonframework.messaging.MessageStream;
+import org.axonframework.messaging.QualifiedName;
+import org.axonframework.modelling.command.StatefulCommandHandlingComponent;
+import org.junit.jupiter.api.*;
+
+import java.util.concurrent.CompletionException;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * Tests whether stateful command handling components can handle commands that target multiple models at the same time.
+ */
+class MultiStateCommandHandlingComponentTest extends AbstractStudentTestsuite {
```

Comment:
Is it MultiState or `MultiEntity`? Probably the second option :)

Reply:
Good catch, adjusting!